vendor/qunit-test-helper-logging-option.js

```diff
@@ -0,0 +1,8 @@
+(function() {
+  if (typeof QUnit !== 'undefined') {
```

Comment:
IMHO, any/all QUnit logic belongs in `ember-qunit`. Let's move this over there, the result will function just the same way but keeps the separation between "worlds" clearer...

Reply:
good point, I will remove that part from the PR

Reply:
done ✅